docs/UserGuide.md

```diff
@@ -161,37 +172,35 @@ Examples:
 Acceptable parameters for INDEX:
 * Only accept **non-negative** int values that are less than the size of the address book. Cannot be 0. Index must be for a contact that has not already been cloned.
 
-![edit format](images/cloneformat.png)
+![clone format](images/cloneformat.png)
 
 Precise expected outputs on success:
 * Message shown to the user: "Cloned Person: X", where X are the details of the person who was cloned.
 * For example, if
 * "Name: John Doe; Phone: 98765432; Email: johnd@example.com; Occupation: Barber; Address: Hougang Avenue 1; AppointmentDate: 2024-02-02; Tags:" was cloned, then the output is
 * "Cloned Person: John Doe; Phone: 98765432; Email: johnd@example.com; Occupation: Barber; Address: Hougang Avenue 1; AppointmentDate: 2024-02-02; Tags: "
 
-![edit format](images/cloneresult.png)
+![clone format](images/cloneresult.png)
 
 Precise expected outputs on failure:
-* When no index, zero or a negative index is entered next to the clone command, the error message
-* "Invalid command format!
-* clone: Clones the person identified by the index number used in the displayed person list.
+* When no index, zero or a negative index is entered next to the clone command, the error message 
+* "Invalid command format! clone: Clones the person identified by the index number used in the displayed person list.
```

Comment:
actually i alr changed this part in my clonecommand pr to match daphnes haha but issokay i can settle the merge conflict when mine is merged